docs/specifications/tests/Consistency-TP/consistency05.md

```diff
@@ -1,49 +1,205 @@
-## CONSISTENCY05: Consistency between glue and authoritative data
+# CONSISTENCY05: Consistency between glue and authoritative data
 
-### Test case identifier
+## Test case identifier
 
-**CONSISTENCY05:** Consistency between glue and authoritative data
+**CONSISTENCY05**
 
-### Objective
+## Objective
 
 For name servers that have IP addresses listed as glue, the IP addresses must
-match the authoritative A and AAAA records for that host. This is an IANA
-requirement (https://www.iana.org/help/nameserver-requirements).
+match the authoritative A and AAAA records for that host. This is an IANA 
+[name server requirement].
+
+The objective of this test is to verify that the glue address records 
+in the delegation are consistent with authoritative data.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+## Ordered description of steps to be taken to execute the test case
+1. Obtain the set of name server names (using [Method2]) and the IP 
+   addresses for those (using [Method4]) for the delegation of 
+   *Child Zone*.
+
+   1. Extract the [in-bailiwick] name servers names and create a set
+      "Delegation Strict Glue" of those, where each name server name 
+      is matched with its IP address or addresses, if available. The 
+      set may be empty.
+
+   2. Extract the [out-of-bailiwick] name servers names and create a set
+      "Delegation Extended Glue" of those, where each name server name 
+      is matched with its IP address or addresses, if available. The 
+      set may be empty.
+
+2. Obtain the set of name server names for the *Child Zone* using
+   [Method2] and [Method3] and extract the [in-bailiwick] name 
+   server names, "IB NS Name Set".
+
+3. Create an empty set of name server name with associated IP address
+   or addresses, "Address Records From Child".
```

Comment:
Do not understand the need of "3".

Reply:
The set is created here and populated below.